src/Bundle/test/src/Tests/Controller/ScienceBookUiTest.php

```diff
@@ -84,6 +84,28 @@ public function it_allows_creating_a_book(): void
         $this->assertSame($newBookAuthorLastName, $book->getAuthorLastName());
     }
 
+    /** @test */
+    public function it_does_not_allow_to_create_a_book_if_there_is_a_validation_error(): void
+    {
+        $newBookTitle = 'The Book of Why';
+        $newBookAuthorLastName = 'Pearl';
+
+        $this->loadFixturesFromFile('science_books.yml');
+
+        $this->client->request('GET', '/science-books/new');
+        $this->client->submitForm('Create', [
+            'science_book[title]' => $newBookTitle,
+            'science_book[author][lastName]' => $newBookAuthorLastName,
+        ]);
+
+        $this->assertResponseCode($this->client->getResponse(), Response::HTTP_UNPROCESSABLE_ENTITY);
```

Comment:
I'm not sure if it is not bc-break and if we didn't send a 400 error. But looking at the code and related PR, I assume we didn't, so I'm fine with this change